kvmagent/kvmagent/plugins/ovs_provision_plugin.py

```
@@ -0,0 +1,1250 @@
+            # MTU comparison (skip rebuilt bonds)
+            for bond_name, bond_spec in desired_bonds.items():
+                if bond_name in rebuilt_bonds or bond_name not in actual_bonds:
+                    continue
+                if bond_spec.mtu is None:
+                    continue
+                for member in actual_bonds[bond_name].members:
+                    if actual_bonds[bond_name].member_mtus.get(member) != bond_spec.mtu:
+                        builder.set_interface(member, 'mtu_request', str(bond_spec.mtu))
+
+            # Refresh config-version on bonds not rebuilt
+            for bond_name, bond_spec in desired_bonds.items():
```

Ruff (0.15.6)

[warning] Loop control variable `bond_spec` not used within loop body

(B007)